gtsfm/run_scene_optimizer.py

```diff
@@ -0,0 +1,49 @@
+import os
```

Comment:
Should we move this file to `gtsfm/runner/run_scene_optimizer.py`? In that case, I can created runners for different modules like front-end only and stuff?

Reply:
i like that idea, i'll move it, one sec

Reply:
done